src/toil/jobStores/googleJobStore.py

```diff
@@ -452,8 +452,8 @@ def _writeFile(self, jobStoreID, fileObj, update=False, encrypt=True):
                 raise NoSuchFileException(jobStoreID)
         blob.upload_from_file(fileObj)
 
-    def _writeString(self, jobStoreID, stringToUpload, **kwarg):
-        self._writeFile(jobStoreID, StringIO(stringToUpload), **kwarg)
+    def _writeBytes(self, jobStoreID, stringToUpload, **kwarg):
```

Comment:
Could typing be added to this function?  Something like  `def _writeBytes(self, jobStoreID: str, stringToUpload: bytes, **kwarg): -> None`.

Reply:
Yep, I'll add that in!